curriculum/js1/2.js

```diff
@@ -7,7 +7,22 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a>b){
+    if(b>c){
```

Comment:
you can reduce this to one statement with `(a>b && a>c)`

Reply:
hey accidently added 2.js, 3.js is the algorithm which needs to be reviewed